atlasdb-dbkvs-tests/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/DbkvsPostgresTestSuite.java

```diff
@@ -41,7 +42,8 @@
         DbkvsPostgresKeyValueServiceTest.class,
         DbkvsPostgresSerializableTransactionTest.class,
         DbkvsPostgresSweeperTest.class,
-        PostgresDbTimestampBoundStoreTest.class
+        PostgresDbTimestampBoundStoreTest.class,
+        PostgresVersionCheckTest.class
```

Comment:
Yeah I thought about that, but didn't find a good existing suite to put it in. Do you have any suggestions? I'm not really familiar with your test infra -- do you scan the classpath for tests, or do you list the suites explicitly? I.e., if I leave that file on its own, will it run? Or do I have to make it part of a suite? And if there is no existing suite, how/where do I add it?